Jenkinsfile

```diff
@@ -47,7 +49,7 @@ pipeline {
             echo 'DRY RUN: Skipping release tag creation'
           }
           else {
-            releaseTag.create(params.RELEASE_NAME)
+            env.GIT_TAG = releaseTag.create(params.RELEASE_NAME)
```

Comment:
I'm not a huge fan of passing around global state with environment variables. Is there some reason why a normal global variable won't suffice? Perhaps I'm misreading the code.

Reply:
Jenkins has some weird rules about variables and their scope based on when they are defined.  I would have to define gitTag outside of the pipeline in order for me to set it in one stage and use it in another.  Since it doesn't have a default value, I'd rather define it as an environment variable which will persist through all stages if it's defined in the script section and not the environment section.